Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -7,7 +7,6 @@ use Slim::Utils::Prefs;
 use Slim::Utils::Strings qw(cstring);
 
 use constant MAX_ALBUMS => 500;
-use constant PRIMARY_ARTIST_ROLES => 'ALBUMARTIST,ARTIST';
```

Comment:
I like constants. What was wrong with that one?

Reply:
It's not constant any more. Depending on the new preference, `COMPOSER` will be appended.